### ISSUE TITLE
should we include The Cow

### ISSUE THREAD

**Issue by cmgchess** [lichess-org/chess-openings#168]
chess.com [has decided](https://youtu.be/htF1uQ1AL3I) to list The Cow in its opening list. should we too?
https://www.chess.com/openings/The-Cow  
https://www.chess.com/openings/The-Cow-2...e5  
https://www.chess.com/openings/The-Cow-2...e5-3.Ne2  
https://www.chess.com/openings/The-Cow-2...e5-3.Ne2-Bd6  
https://www.chess.com/openings/The-Cow-2...e5-3.Ne2-Bd6-4.Ng3  
https://www.chess.com/openings/The-Cow...3.Ne2-Bd6-4.Ng3-Nf6  
https://www.chess.com/openings/The-Cow...3.Ne2-Bd6-4.Ng3-Nf6-5.Nd2  
https://www.chess.com/openings/The-Cow...4.Ng3-Nf6-5.Nd2-O-O  
https://www.chess.com/openings/The-Cow-Bull-Variation

**Comment by dSupertramp** [comment]
Why not?

**Comment by HaonRekcef** [comment]
One reason that speaks against it is that we would have to cover a lot of move orders, as black can basically play any move

**Comment by HaonRekcef** [comment]
One reason that speaks against it is that we would have to cover a lot of move orders, as black can basically play any move.

**Comment by cmgchess** [comment]
> One reason that speaks against it is that we would have to cover a lot of move orders, as black can basically play any move

i think the moves are fixed here if going with the order e3 d5 d3

**Comment by HaonRekcef** [comment]
Is the cow not defined by the knights going to b3 and g3, or is it already enough to go with e3 d3. What about black playing the cow?

**Comment by ryathom57** [comment]
If this is added, and I see the Alien Gambit is added, should other trendy openings be added to, such as the Wagon Gambit?

**Comment by chao-mu** [comment]
It's a slice of chess history, whether silly and ephemeral or not.

**Comment by Mega-Noob-1337** [comment]
I see 3 solutions to this:
1. Top engine moves
2. Symmetrical
3. Ask the inventor for black's response

**Comment by Mega-Noob-1337** [comment]
Alright it was very silly of me to expect any kind of respone to an issue that was abandoned for 2 years

**Comment by Mega-Noob-1337** [comment]
I'll just go and make a pull request fixing this

**Comment by cakiki** [comment]
I don't think we should. The criteria for inclusion aren't set anywhere, but generally an opening should be:
- played regularly
- seriously played by world class players / titled players / opening specialists
- be covered in the literature

The Cow doesn't really fulfill those in a way that warrants its inclusion.
